language/move-lang/tests/move_check/typing/borrow_local_temp_resource.exp

```diff
@@ -1,28 +1,20 @@
 error: 
 
-   ┌── tests/move_check/typing/borrow_local_temp_resource.move:5:14 ───
+   ┌── tests/move_check/typing/borrow_local_temp_resource.move:6:10 ───
    │
- 5 │       fun t0() {
-   │ ╭──────────────^
- 6 │ │         &R{};
- 7 │ │         &mut R{};
- 8 │ │     }
-   │ ╰─────^ Invalid return
+ 7 │         &mut R{};
+   │                 ^ Invalid return
    ·
  6 │         &R{};
    │          --- The resource is created but not used. The resource must be consumed before the function returns
    │
 
 error: 
 
-   ┌── tests/move_check/typing/borrow_local_temp_resource.move:5:14 ───
+   ┌── tests/move_check/typing/borrow_local_temp_resource.move:7:14 ───
    │
- 5 │       fun t0() {
-   │ ╭──────────────^
- 6 │ │         &R{};
- 7 │ │         &mut R{};
- 8 │ │     }
-   │ ╰─────^ Invalid return
+ 7 │         &mut R{};
+   │                 ^ Invalid return
```

Comment:
A little confusing still, but I think its at least better than before. Progress!

Reply:
Yes, I think this is much better since it calls attention to the trailing semicolon, which should probably just be deleted.